doc/guides/bluetooth/autopts/autopts-win10.rst

```diff
@@ -0,0 +1,393 @@
+
+.. image:: install_pts_drivers.png
+   :height: 250
+   :width: 850
+   :align: center
+
+Setup Zephyr project for Windows
+=================================
+
+Setup from Zephyr site https://docs.zephyrproject.org/latest/getting_started/index.html:
+
+Open Git Bash and go to home:
+
+.. code-block::
+
+    cd ~
+
+Install west:
+
+.. code-block::
+
+    pip3 install west
+
+Get the Zephyr source code:
+
+.. code-block::
+
+    west init zephyrproject:
+
+Go into freshly created folder:
+
+.. code-block::
+
+    cd zephyrproject
+
+Run:
+
+.. code-block::
+
+    west update
+
+Export a Zephyr CMake package. This allows CMake to automatically load
+boilerplate code required for building Zephyr applications:
+
+.. code-block::
+
+    west zephyr-export
+
+Zephyr’s scripts/requirements.txt file declares additional Python
+dependencies. Install them with pip:
+
+.. code-block::
+
+    pip3 install -r ~\zephyrproject\zephyr\scripts\requirements.txt
+
+Setup WSL1 with Ubuntu 20.4
+============================
+
+Setup Install Ubuntu 20.4 on `WSL1 <https://docs.microsoft.com/en-us/windows/wsl/install-win10>`_.
+Open PowerShell as Administrator and run:
+
+.. code-block::
+
+   dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart
+
+Restart Windows. After restart, open Microsoft Store and install Ubuntu 20.4 LTS.
+
+.. image:: install_ubuntu_on_wsl.png
+   :height: 350
+   :width: 500
+   :align: center
+
+Run Ubuntu. You will be asked to create a user account and password:
+
+.. image:: ubuntu_first_launch.png
+   :height: 150
+   :width: 800
+   :align: center
+
+When finished, run commands:
+
+.. code-block::
+
+    sudo apt update
+    sudo apt upgrade
+
+Install python3:
+
+.. code-block::
+
+    sudo apt install python3
+
+Install pip:
+
+.. code-block::
+
+    sudo apt install python3-pip
+
+Install west:
+
+.. code-block::
+
+    pip3 install --user -U west
+
+Export local bin to PATH:
+
+.. code-block::
+
+    echo 'export PATH=~/.local/bin:"$PATH"' >> ~/.bashrc
+
+Reload PATH:
+
+.. code-block::
+
+    source ~/.bashrc
+
+Install cmake:
+
+.. code-block::
+
+    sudo apt install cmake
+
+Go to your zephyrproject:
+
+.. code-block::
+
+    cd /mnt/c/Users/Codecoup/zephyrproject
+
+and then run:
+
+.. code-block::
+
+    west zephyr-export
+    pip3 install --user wheel
+    pip3 install --user -r /mnt/c/Users/codecoup/zephyrproject/zephyr/scripts/requirements.txt
+
+Check if all modules have been installed:
+
+.. code-block::
+
+    pip3 list
+
+If modules still will be missing, just install them with:
+
+.. code-block::
+
+    pip3 install <module_name>
+
+Install Ninja:
+
+.. code-block::
+
+    pip3 install ninja
+
+Go to home:
+
+.. code-block::
+
+    cd ~
+
+and download a toolchain installer:
+
+.. code-block::
+
+    wget https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.12.3/zephyr-sdk-0.12.3-x86_64-linux-setup.run
```

Comment:
You're referencing 0.12.4 in another section of this PR. Are they supposed to be the same? Again, my concern is that these references will become outdated every time a new SDK version is released.